src/view/MixerElementVolumeWindow.vala

```diff
@@ -0,0 +1,110 @@
+/*
+ * brickman -- Brick Manager for LEGO MINDSTORMS EV3/ev3dev
+ *
+ * Copyright (C) 2016 Kaelin Laundry <wasabifan@outlook.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * MixerElementVolumeWindow.vala - Allows control of mixer element volume
+ */
+
+using Ev3devKit;
+using Ev3devKit.Ui;
+
+namespace BrickManager {
+    public class MixerElementVolumeWindow : BrickManagerWindow {
+        public signal void volume_up ();
+        public signal void volume_down ();
+        public signal void mute ();
+
+        IMixerElementViewModel _current_element;
+        bool _show_element_details = true;
+
+        Ui.Label element_label;
+
+        public MixerElementVolumeWindow () {
+            title = "Volume control";
```

Comment:
The title boxes don't handle strings that are long enough to wrap to multiple lines very well, so I'd rather not risk that odd behavior.